lib/identity_cache/cached/attribute.rb

```diff
@@ -47,6 +47,15 @@ def expire(record)
         end
       end
 
+      def expire_by_key_value(key_values)
+        key_fields.each do |key| 
+          return unless key_values.key?(key)
```

Comment:
Should we raise instead?

Reply:
I also thought about that, but what happens if you have:
```ruby
cache_attribute :tracked, :id
cache_attribute :tracked, :sku
cache_attribute :tracked, :another_thing
```
And for whatever reason you want to expire just `id` and `sku`, then this would raise. Why would you want to do that? I don't really believe there is a good reason, but I am trying to make it as flexible as possible.

That being said, we could make this method being very specific to expire ALL of the indexes and the contract being you have to provide all the different indexes.

Reply:
I was thinking that in most cases you want to expire all the caches, and perhaps we should optimize for ensuring we don't forget one.

Reply:
We could definitely raise an error then..